inst/sql_queries/initialize_metric_table.sql

```diff
@@ -14,4 +14,5 @@ VALUES
 ('dependencies',        'Dependencies',      'Number of Package Dependencies',   0, 0, 'maintenance', 1),  
 ('reverse_dependencies','Reverse Dependencies', 'Number of Reverse Dependencies', 0, 0, 'community', 1),
 ('covr_coverage',       'Test Coverage',     '% of objects tested',     1, 0, 'maintenance', 1),
-('downloads_1yr', 'Package Downloads', 'Number of downloads in last 12 months', 0, 0, 'community', 1);
\ No newline at end of file
+('downloads_1yr', 'Package Downloads', 'Number of downloads in last 12 months', 0, 0, 'community', 1),
+('suggests', 'Suggests', 'Number of Package Suggests', 0, 0, 'dependency', 0);
```

Comment:
Note: whenever you change the database in any way, you now need to run `creating_testing_databases.R` in the `data-raw` folder thanks to #629 . But that is a pretty recent change so I wouldn't expect you to be aware of it yet. Jeff did add a new bullet point to the [PR checklist](https://github.com/pharmaR/riskassessment/wiki/Developer-Pull-Request-workflows#pr-checklist), but @Jeff-Thompson12, could you also announce this new workflow and give a quick recap of what it does on slack so everyone is aware?

Reply:
Also, I think this is the first non `riskmetric` metric we are actually storing in the database. All the other ones included in the app are calculated on the fly.

Reply:
OK
added the materialSwitch in place of an action button
and ran `creating_testing_databases.R`